components/MapSection/MapSection.tsx

```diff
@@ -69,10 +70,18 @@ const MapSection: FunctionComponent<MapSectionProps> = ({ onChangeLocation, ...p
   const onOptionsChange = (optns: SpotlightOptions): void => {
     setQuery(optns, activeLocation && [activeLocation]);
     setOptions(optns);
+    addEvent('mapsectionOptionsChanged', {
```

Comment:
proper camel case ... u know I'm a sucker for such things... consistency.

Reply:
Updated to camel case